src/_base/harness/config/events.yml

```diff
@@ -3,10 +3,13 @@ after('harness.install'): |
   #!bash
   ws enable
 
-after('harness.refresh'): |
-  #!bash(workspace:/)|@
-  run docker-compose -p @('namespace') stop
-  passthru "docker-compose config --services | grep -v php-fpm | xargs docker-compose pull"
-  passthru "docker-compose config --services | grep -v cron | grep -v job-queue-consumer | grep -v jenkins-runner | xargs docker-compose build --pull"
-  passthru "docker-compose config --services | grep -v cron | grep -v job-queue-consumer | grep -v jenkins-runner | xargs docker-compose up -d"
-  run docker-compose -p @('namespace') up -d --build
+after('harness.refresh'):
+  env:
```

Comment:
Not sure this works for events

Reply:
buggy in the first place as well as the passthru's didn't set -p

Reply:
True!

Reply:
David says it works for these